__tests__/app.test.js

```diff
@@ -337,6 +337,28 @@ describe('POST /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('DELETE /api/comments/comment_id', () => {
+  test('204: Responds with status code after deleting given comment_id from table', () => {
+    return request(app).delete('/api/comments/2').expect(204);
+  });
```

Comment:
Well done, on a delete endpoint we can't send anything on body so just the 204 is enough :)